app/src/main/java/by/academy/jc/zaikouskaya/AnagramUtils.java

```diff
@@ -0,0 +1,45 @@
+package by.academy.jc.zaikouskaya;
+
+import org.jetbrains.annotations.NotNull;
```

Comment:
Use `javax.validation.NotNull` instead please.

See Joshua Bloch programming through interface,

Reply:
My IDE can't find this library.